src/leiningen/i18n/bin/update-pot.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -eo pipefail
+
+repo_dir="$(realpath "$(dirname "$0")/../../..")"
+pot_file="$repo_dir/locales/messages.pot"
```

Comment:
I can't think of anything further to do to future-proof it past putting it in the variable here so it only has to be redefined in one place. I think anything more would depend on how exactly they're going to be renamed, which we don't know yet.